src/ArborX_BruteForce.hpp

```diff
@@ -58,10 +58,11 @@ class BasicBruteForce
   KOKKOS_FUNCTION
   bounding_volume_type bounds() const noexcept { return _bounds; }
 
-  template <typename ExecutionSpace, typename Predicates, typename Callback,
-            typename Ignore = int>
+  template <typename ExecutionSpace, typename Predicates, typename Callback>
   void query(ExecutionSpace const &space, Predicates const &predicates,
-             Callback const &callback, Ignore = Ignore()) const;
+             Callback const &callback,
+             Experimental::TraversalPolicy const &policy =
+                 Experimental::TraversalPolicy()) const;
```

Comment:
`Experimental::TraversalPolicy` makes absolutely no sense for `BruteForce`.  This was literally the point of that `Ignore` parameter.

Reply:
It just doesn't make much sense to me to have a user-exposed template parameter that allows passing in arbitrary types that are entirely ignored.